github-runners/k8s/templates/base/runner.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: actions.summerwind.dev/v1alpha1
+kind: RunnerDeployment
+metadata:
+ name: k8s-github-runner
+ namespace: actions-runner-system
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: k8s-github-runner
+  template:
+    metadata:
+      labels:
+        app: k8s-github-runner
+    spec:
+      repository: https://github.com/subspace/subspace-cli
```

```suggestion
      repository: https://github.com/subspace/<REPOSITORY>
```

Prefer  not to default in template as its easier to assume cli is a requirement here when I looked at it initially